src/tiler/src/util/layer-filter.js

```diff
@@ -16,6 +16,65 @@ const parsePromise = xmlString => new Promise((resolve, reject) => {
         else resolve(result)
     })
 })
+/**
+ * Structures sql based on existing query and json options
+ */
+const structureQuery = (existingQuery, currentLayer) => {
+    // Base of the query, wrapping pre-existing query
+    let query = `SELECT * FROM ${existingQuery} WHERE`
+
+    // Add each filter to the query string
+    currentLayer.filters.forEach((filterObj, i) => {
+        // Determine comparison mode (if more than one parameter)
+        let mode = ''
+        if (i > 0) {
+            if (currentLayer.mode) {
+                switch (currentLayer.mode.toUpperCase()) {
+                case 'OR':
+                    mode = 'OR'
+                    break
+                default:
+                    mode = 'AND'
+                    break
+                }
+            } else {
+                mode = 'AND'
+            }
+        }
+
+        // Determine comparison operator
+        let op = '='
+        if (filterObj.op) {
+            switch (filterObj.op.toUpperCase()) {
+            case 'LIKE':
+            case 'IS':
+            case 'IS NOT':
+            case 'IS DISTINCT FROM':
+            case 'IS NOT DISTINCT FROM':
+                op = ` ${filterObj.op.toUpperCase()} `
+                break
+            case '<':
+            case '>':
+            case '<=':
+            case '>=':
+            case '<>':
+            case '!=':
+                /* eslint-disable-next-line prefer-destructuring */
+                op = filterObj.op
+                break
+            default:
+                op = '='
+                break
+            }
+        }
+
+
+        query += `${mode ? ` ${mode}` : ''} ${filterObj.col}${op}'${filterObj.val}'`
```

Comment:
I ditched the idea of sanitizing the inputs early on (when the filtering paradigm was still "just put the whole query in a url parameter) but forgot to re-introduce it now that the only variables in play are the column name and value! I'll get on that. Regarding the error tile issue: I've removed them in #95 and also filtered caught errors so that the Postgis ones never make it out of the back end.